### PR TITLE
result -> results

### PR DESCRIPTION
**Pull request by arnetheduck**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -12847,17 +12847,21 @@
...
+  },
+  {
+    "name": "results",
+    "url": "https://github.com/arnetheduck/nim-results",
     "method": "git",
     "tags": [
       "library",
       "result",
       "errors",
       "functional"
```
Comment (Yardanico) [inline packages.json:12860]: Just a thought - maybe also add the tag `option` so there's a higher possibility of people finding the package?

Comment (arnetheduck) [reply packages.json]: 6e9e61c8fe8906c350ace4038836be22ef05ca39

**Comment by ringabout** [comment]
Weird, it breaks the Nim CI, probably some hack in the testament.

**Comment by arnetheduck** [comment]
> Weird, it breaks the Nim CI, probably some hack in the testament.

`results` was previously available as `import result` (singular) - it's been deprecated for years but was removed as part of the rename

**Pull request merged**